sandbox-backend/src/main/java/com/exadel/sandbox/team5/configs/security/util/JwtUtil.java

```diff
@@ -0,0 +1,58 @@
+package com.exadel.sandbox.team5.configs.security.util;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.stereotype.Component;
+
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+
+@Component
+public class JwtUtil {
+
+    //    @Value("${app.jwtSecret}")
```

Comment:
You can use @Value with default value, for example `@Value("${app.jwtSecret:salt}`

Reply:
how do I know salt? I must use this jwtSecret phrase in BCrypt generator that change password  in test DB

Reply:
You can put it into application.yml and use in code with @Value annotation